src/ai/backend/client/cli/scp.py

```diff
@@ -0,0 +1,48 @@
+import subprocess
+import click
+from . import main
+
+
+@main.command()
+@click.argument("sess_name",  type=str, metavar='SESS_NAME')
+@click.argument("src",  type=str, metavar='SRC')
+@click.argument("dest",  type=str, default="work@localhost", metavar='DEST')
+@click.option('-p', '--port',  type=str, metavar='PORT', default=9922,
+              help="host port number")
```

Comment:
The help text should be more descriptive, indicating that the port number is for localhost.